.github/ISSUE_TEMPLATE/release.md

```diff
@@ -31,15 +31,25 @@ The following checks can be performed after we have frozen our release candidate
 - [ ] Complete the following [manual QA workflow](https://hackmd.io/TbFmorG2RnOPmLuFcg9JOQ?view).
 - [ ] Verify Polkadot JS API are up to date with the latest
     runtime changes.
-- [ ] Execute runtime upgrade to Baikal and verify network stability.
-- [ ] Execute runtime upgrade to Rococo and verify network stability.
+- [ ] Execute runtime upgrade to Baikal relay and verify network stability.
+- [ ] Execute runtime upgrade to Calamari @ Baikal and verify network stability.
+- [ ] Execute runtime upgrade to Calamari @ Moonbase-Relay and verify network stability.
+- [ ] Execute runtime upgrade to Dolphin @ Baikal and verify network stability.
+- [ ] Check that the new runtime versions have [burned-in](#burn-in) without issue for at least 12 hours.
 - [ ] Prepare a governance post and submit to our forum with description and motivation for changes.
 
+Note: Usually update client first then runtime.
```

Comment:
I think we should do this from what a user will do.
Many ppl will upgrade after the RT is already live ( except overzealous collator runners )
We should test for that case

Reply:
I'm not sure if most of our node runners wait for the RT before updating their client. We should double check that, cuz we've been kind of aiming for the opposite. At least internally.

Reply:
That's why i'm calling them overzealous

The normal polkadot usecase is that  you rarely update the client and simply use the updated runtimes that get passed to you from the chain.
There's rarely a reason to even upgrade your client at all, so i'd say no matter what our collator runners do currently ( maybe we should tell them to upgrade only after the chain vote) we should do runtime first

Reply:
I think the other reason for that ordering was that historically there have been a lot more breaking client changes

Reply:
right, everytime we fiddle around with RPC or consensus we need to force a client upgrade.
but even in these cases there's no reason to prefer one sequence over the other
- client first and
- runtime first
must both work. It's just that in these cases a client update is mandatory, it doesn't matter tho in which order.
In the RPC case, if you up the client and ppl start using it the RPCs will just fail because the runtime isnt there to handle the request, and if you upgrade the runtime first, nobody will be able to use it until they update client.

Reply:
The point is more often than not we'll have to change the process. Also this is what the CI is doing, and actually that's why we ended up with client first, because the CI was failing a lot due to frequent client breaking changes